.gitignore

```diff
@@ -27,3 +27,16 @@ config*.json
 # Pyinstaller files
 build
 dist
+
+# Environments
+.env
+.venv
+env/
+venv/
+ENV/
+env.bak/
+venv.bak/
+
+# Memory dump files
+*.dmp
```

Comment:
This isn't standardized, but I'll allow it since it's common, I just don't want to be adding every file extension someone thinks should be ignored.